lib/active_record/mass_assignment_security/association_relation.rb

```diff
@@ -7,19 +7,16 @@ class AssociationRelation
       undef :create!
 
       def build(attributes = nil, options = {}, &block)
-        block = _deprecated_scope_block("new", &block)
-        scoping { @association.build(attributes, options, &block) }
+        @association.build(attributes, options, &block)
```

Comment:
Does this change behaviour for Rails 5.2 and below?

Reply:
It shouldn't as far as I can tell. There's already a test for this that can be found at `test/mass_assignment_security/association_relation_test.rb` and it passes the on 5.0, 5.1 and 5.2. But, just to be sure, I went back on our app that we've been upgrading from 5.1 all the way to 6.1, and using this version I can say that all our existing test passes.